### PR TITLE
refactor(WIP): mycollection sticky tabs migration #doNotMerge

### PR DESCRIPTION
**Pull request by gkartalis**
This PR resolves [MOPLAT-786] 

### PR Checklist

- [ ] I have tested my changes on **iOS** and **Android**.
- [ ] I hid my changes behind a **[feature flag]**, or they don't need one.
- [ ] I have included **screenshots** or **videos**, or I have not changed the UI.
- [ ] I have added **tests**, or my changes don't require any.
- [ ] I added an **[app state migration]**, or my changes do not require one.
- [ ] I have documented any **follow-up work** that this PR will require, or it does not require any.
- [ ] I have added a **changelog entry** below, or my changes do not require one.

### To the reviewers 👀

- [ ] I would like **at least one** of the reviewers to **run** this PR on the simulator or device.

<details><summary>Changelog updates</summary>

#### Cross-platform user-facing changes

-

#### iOS user-facing changes

-

#### Android user-facing changes

-

#### Dev changes

-

</details>

Need help with something? Have a look at our [docs], or get in touch with us.

[app state migration]: ../blob/main/docs/adding_state_migrations.md
[feature flag]: ../blob/main/docs/developing_a_feature.md
[docs]: ../blob/main/docs/README.md

[MOPLAT-786]: https://artsyproduct.atlassian.net/browse/MOPLAT-786?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/app/Components/Tabs/TabsContainer.tsx [hunk 1]
```diff
@@ -58,9 +58,6 @@ export const TabsContainer: React.FC<TabsContainerProps> = ({
               activeColor={color("onBackground")}
               inactiveColor={color("onBackgroundMedium")}
               labelStyle={{ marginTop: 0 }} // removing the horizonal margin from the lib
-              tabStyle={{
-                marginHorizontal: 10,
-              }} // adding the margin back here
```
Comment (gkartalis) [inline src/app/Components/Tabs/TabsContainer.tsx:63]: Looks like this is only causing issues and not doing anything.

i.e. in My Collection tabs the `My Collection` title of the tab is broken in two lines if we leave this in even in bigger devices. 

Without it it just works ⭐️

File: src/app/Scenes/Partner/Components/PartnerOverview.tsx [hunk 2]
```diff
@@ -16,9 +15,9 @@ export const PartnerOverview: React.FC<{
 
   if (!aboutText && !cities) {
     return (
-      <StickyTabPageScrollView>
+      <TabScrollView>
```
Comment (gkartalis) [inline src/app/Scenes/Partner/Components/PartnerOverview.tsx:18]: not related with my collection but forgot to address it before so refactoring here

**Comment by gkartalis** [comment]
@MounirDhahri closing this after our sync but we will have the diff in case we need it

**Comment by MounirDhahri** [comment]
Sounds good

File: src/app/Components/Tabs/TabsContainer.tsx (hunk 1 above)
Comment (gkartalis) [reply src/app/Components/Tabs/TabsContainer.tsx:63]: cc @damassi that was a small gotcha.

Comment (gkartalis) [reply src/app/Components/Tabs/TabsContainer.tsx:63]: yep this is the culprit for sure

**Pull request closed without merging**